src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -211,6 +217,36 @@ function BulkEmailForm(props) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isScheduled, editor.editMode, editor.isLoading, editor.errorRetrievingData, editor.formComplete]);
 
+  /*
+   This will be checking if there are emails added to the emailLearnersList state
+   if so, we will delete emailRecipients "learners" because that is for all learners
+   if not, we will delete the individual-learners-emails from emailRecipients because of we won't use the emails
+  */
+  useEffect(() => {
+    if (emailLearnersList.length && !editor.emailRecipients.includes('individual-learners-emails')) {
+      dispatch(addRecipient('individual-learners-emails'));
+      if (editor.emailRecipients.includes('learners')) {
+        dispatch(removeRecipient('learners'));
+      }
+    } else if (!emailLearnersList.length && editor.emailRecipients.includes('individual-learners-emails')) {
+      dispatch(removeRecipient('individual-learners-emails'));
+    }
+  }, [dispatch, editor.emailRecipients, emailLearnersList]);
+
+  // When the user selects an email from input autocomplete list
+  const handleEmailLearnersSelected = (emailSelected) => {
+    const [firstItem] = emailSelected;
+    if (firstItem) {
+      setEmailLearnersList([...emailLearnersList, firstItem]);
+    }
+  };
+
+  // To delete an email from learners list, that list is on the bottom of the input autocomplete
+  const handleDeleteEmailLearnerSelected = (idToDelete) => {
+    const setEmailLearnersListUpdated = emailLearnersList.filter(({ id }) => id !== idToDelete);
+    setEmailLearnersList(setEmailLearnersListUpdated);
+  };
```

Comment:
Same question here about not modifying the MFE since that means more maintenance efforts